lib/matplotlib/patches.py

```diff
@@ -1840,7 +1840,7 @@ def set_width(self, width):
         ----------
         width : float
         """
-        if min(self.a, self.b) <= width:
+        if width <= min(self.a, self.b) is False:
```

Comment:
The comparison plus `is False` is a bit odd; either use `not()` or invert the condition.

Reply:
```suggestion
        if width > min(self.a, self.b):
```